scripts/talos/bumpSimorghPackages/index.js

```diff
@@ -0,0 +1,55 @@
+const getRemoteFile = require('../../utilities/getRemoteFile');
+const createRemoteBranch = require('../../utilities/createRemoteBranch');
+const createPullRequest = require('../../utilities/createPullRequest');
+const getPackageVersion = require('../../utilities/getPackageVersion');
+const getPackageNames = require('../createPullRequest/getPackageNames');
+const getChangelogHead = require('../getChangelogHead');
+const getUpdates = require('./getUpdates');
+const updatePackageLock = require('./updatePackageLock');
+const updatePackageJson = require('./updatePackageJson');
+
+const path = 'package.json';
+const repoName = 'simorgh';
+
+const getSimorghPullRequestBody = updates => {
+  return updates
+    .map(update => `${update}\n\n${getChangelogHead(update)}`)
```

Comment:
Why have we got 2 ways of creating pull request bodies? (psammead and simorgh prs) when they can be the same?

Reply:
The two repositories are different in structure. Psammead contains multiple packages, while Simorgh is a single repository. So we need to keep the bodies separate.